aws-fargate-terraform/ecs_customer_service.tf

```diff
@@ -0,0 +1,65 @@
+resource "aws_ecs_service" "svc_customer" {
+  name            = "svc-customer"
+  launch_type     = "FARGATE"
+  cluster         = "${aws_ecs_cluster.cluster.id}"
+  task_definition = "${aws_ecs_task_definition.task-customer.arn}"
+  desired_count   = 1
+
+  depends_on = [
+    "aws_iam_role_policy.ecs_service_role_policy",
+  ]
+
+  lifecycle {
+    create_before_destroy = true
+
+    ignore_changes = [
+      "desired_count",
+    ]
+  }
+
+  network_configuration {
+    security_groups = [
+      "${aws_security_group.sg_customer.id}",
+    ]
+
+    subnets = [
+      "${aws_subnet.public-subnet.id}",
+      "${aws_subnet.public-subnet1.id}",
+    ]
+
+    assign_public_ip = true
+  }
+}
+
+data "template_file" "customer_task_definition" {
+  template = "${file("${path.module}/ecs_customer_definition.json")}"
+
+  vars {
+    db_url             = "jdbc:mysql://${aws_db_instance.mysql_instance.endpoint}/${aws_db_instance.mysql_instance.name}"
+    db_pwd             = "${aws_db_instance.mysql_instance.password}"
+    db_user            = "${aws_db_instance.mysql_instance.username}"
+    zookeeper_id = "${element(split(",", aws_msk_cluster.eventuate.zookeeper_connect_string), 0)}"
```

Comment:
Rename `zookeeper_id` to zookeeper_connection_string - no need to split and take the first element.